boot/bootutil/include/bootutil/crypto/ecdsa_p256.h

```diff
@@ -31,14 +31,125 @@
 
 #if defined(MCUBOOT_USE_MBED_TLS)
     #include <mbedtls/ecdsa.h>
-    #include "bootutil/crypto/common.h"
     #define BOOTUTIL_CRYPTO_ECDSA_P256_HASH_SIZE (4 * 8)
+    /* Indicate to the caller that the verify function needs the raw ASN.1
+    * signature, not a decoded one. */
+    #define MCUBOOT_ECDSA_NEED_RAW_ASN1_SIG
```

Comment:
What's the reason for the renaming? (from MCUBOOT_ECDSA_NEED_ASN1_SIG)
For me, the "RAW" prefix suggests a "raw" byte string.